flake.lock

```diff
@@ -38,6 +38,22 @@
         "type": "github"
       }
     },
+    "flake-compat_2": {
```

Comment:
Make this follow `flake-compat`?

Reply:
I tried my best but didn't manage to. I'll merge as-is and we'll figure it out in a new PR. Sorry.